Bio/PDB/Vector.py

```diff
@@ -10,194 +10,8 @@
-
-    Example:
-
-        >>> m=rotaxis(pi, Vector(1, 0, 0))
-        >>> rotated_vector=any_vector.left_multiply(m)
-
-    @type theta: float
-    @param theta: the rotation angle
-
-
-    @type vector: L{Vector}
-    @param vector: the rotation axis
-
-    @return: The rotation matrix, a 3x3 Numeric array.
-    """
-    vector = vector.copy()
-    vector.normalize()
-    c = numpy.cos(theta)
-    s = numpy.sin(theta)
-    t = 1 - c
-    x, y, z = vector.get_array()
-    rot = numpy.zeros((3, 3))
-    # 1st row
-    rot[0, 0] = t * x * x + c
-    rot[0, 1] = t * x * y - s * z
-    rot[0, 2] = t * x * z + s * y
-    # 2nd row
-    rot[1, 0] = t * x * y + s * z
-    rot[1, 1] = t * y * y + c
-    rot[1, 2] = t * y * z - s * x
-    # 3rd row
-    rot[2, 0] = t * x * z - s * y
-    rot[2, 1] = t * y * z + s * x
-    rot[2, 2] = t * z * z + c
-    return rot
-
-
-rotaxis = rotaxis2m
-
-
-def refmat(p, q):
-    """Return a (left multiplying) matrix that mirrors p onto q.
-
-    Example:
-        >>> mirror=refmat(p, q)
-        >>> qq=p.left_multiply(mirror)
-        >>> print(q)
-        >>> print(qq) # q and qq should be the same
-
-    @type p,q: L{Vector}
-    @return: The mirror operation, a 3x3 Numeric array.
-    """
-    p.normalize()
-    q.normalize()
-    if (p - q).norm() < 1e-5:
-        return numpy.identity(3)
-    pq = p - q
-    pq.normalize()
-    b = pq.get_array()
-    b.shape = (3, 1)
-    i = numpy.identity(3)
-    ref = i - 2 * numpy.dot(b, numpy.transpose(b))
-    return ref
-
-
-def rotmat(p, q):
-    """Return a (left multiplying) matrix that rotates p onto q.
-
-    Example:
-        >>> r=rotmat(p, q)
-        >>> print(q)
-        >>> print(p.left_multiply(r))
-
-    @param p: moving vector
-    @type p: L{Vector}
-
-    @param q: fixed vector
-    @type q: L{Vector}
-
-    @return: rotation matrix that rotates p onto q
-    @rtype: 3x3 Numeric array
-    """
-    rot = numpy.dot(refmat(q, -p), refmat(p, -p))
-    return rot
-
-
-def calc_angle(v1, v2, v3):
-    """Calculate the angle between 3 vectors representing 3 connected points.
-
-    @param v1, v2, v3: the tree points that define the angle
-    @type v1, v2, v3: L{Vector}
-
-    @return: angle
-    @rtype: float
-    """
-    v1 = v1 - v2
-    v3 = v3 - v2
-    return v1.angle(v3)
-
-
-def calc_dihedral(v1, v2, v3, v4):
-    """Calculate the dihedral angle between 4 vectors representing 4 connected points.
-
-    The angle is in ]-pi, pi].
-
-    @param v1, v2, v3, v4: the four points that define the dihedral angle
-    @type v1, v2, v3, v4: L{Vector}
-    """
-    ab = v1 - v2
-    cb = v3 - v2
-    db = v4 - v3
-    u = ab ** cb
-    v = db ** cb
-    w = u ** v
-    angle = u.angle(v)
-    # Determine sign of angle
-    try:
-        if cb.angle(w) > 0.001:
-            angle = -angle
-    except ZeroDivisionError:
-        # dihedral=pi
-        pass
-    return angle
-
-
 class Vector(object):
-    """3D vector."""
+    "3D vector"
```

Comment:
This is why the style check is failing under TravisCI, lots of entries like this from the ``pydocstyle`` tool;

```
D300: Use """triple double quotes""" (found '-quotes)
```

You can install this code checking tool with ``pip install pydocstyle``